apps/prairielearn/src/pages/partials/submission.ejs

```diff
@@ -73,7 +73,14 @@
           <% } %>
         </span>
       </div>
-      <small>Submitted at <%- submission.formatted_date %></small>
+      <span class="small">
+        <% if (authz_data.user.uid == submission.uid) { %>
+          <span class="pl-submission-attribution">You</span> submitted
+        <% } else { %>
+          Submitted by <%= authz_data.user.name %> (<span class="pl-submission-attribution"><%= submission.uid %></span>)
```

Comment:
`authz_data.user.name` is the effective user's name, not the user associated to the submission.

Reply:
Ahh, is the user associated to the submission just `submission.user`?

Reply:
The user associated to the submission is what is being queried in your change to `question.sql`. I don't think this is queried yet, you'd need to add it to the query.

Reply:
We're no longer showing the name.